src/main/java/uiPages/LoginPage.java

```diff
@@ -32,18 +32,9 @@
   @FindBy(xpath = "//p[contains(@class, 'lost_password')]/a")
   private WebElement lostPasswordLink;
 
-  @FindBy(xpath = "//ul[@class='woocommerce-error']/li")
+  @FindBy(xpath = "//ul[@class='woocommerce-error']")
```

Comment:
Not sure I understand why you removed the "li" html element from the xpath. The message is located within the "li" html element, not directly in the "ul" html element.